bigframes/pandas/__init__.py

```diff
@@ -486,6 +487,8 @@ def read_gbq(
     index_col: Iterable[str] | str = (),
     col_order: Iterable[str] = (),
     max_results: Optional[int] = None,
+    columns: Iterable[str] = (),
```

Comment:
Please pull the `columns` change out into a separate PR to be reviewed outside of the filters change. IMO, `columns` and `col_order` are redundant and both should select a subset of columns. We need to keep both for compatibility see: https://github.com/googleapis/python-bigquery-pandas/pull/701

Reply:
Done, removed.